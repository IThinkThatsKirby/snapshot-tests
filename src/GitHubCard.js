import * as React from 'react';
import Card from '@mui/material/Card';
import CardActions from '@mui/material/CardActions';
import CardContent from '@mui/material/CardContent';
import Button from '@mui/material/Button';
import Typography from '@mui/material/Typography';

export default function GitHubCard() {
	return (
		<Card sx={{ minWidth: 275 }}>
			<CardContent>
				<Typography sx={{ fontSize: 42 }} color="red">
					IThinkThatsKirby
				</Typography>
			</CardContent>
			<h1>PICTURE OF ME GOES HERE.jpeg</h1>
			<Typography sx={{ textAlign: 'right', textTransform: 'uppercase' }}>
				I am really good at front end.
			</Typography>
			<CardActions>
				<CardContent>
					<Button size="small" href="https://github.com/IThinkThatsKirby">
						Visit my Git hub HERE
					</Button>
				</CardContent>
			</CardActions>
		</Card>
	);
}
