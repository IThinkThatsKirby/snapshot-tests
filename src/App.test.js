import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';
import GitHubCard from './GitHubCard';
import Typography from '@mui/material/Typography';

test('renders stuff', () => {
	render(<App />);
	const pic = renderer.create(<GitHubCard />).toJSON();
	expect(pic).toMatchSnapshot();
});
test('rendersName', () => {
	render(<App />);
	const name = screen.getByText('IThinkThatsKirby');
	expect(name).toBeInTheDocument();
});
