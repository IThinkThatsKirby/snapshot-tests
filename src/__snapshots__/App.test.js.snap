// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders stuff 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-pwngrv-MuiPaper-root-MuiCard-root"
>
  <div
    className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-a08tdl-MuiTypography-root"
    >
      IThinkThatsKirby
    </p>
  </div>
  <h1>
    PICTURE OF ME GOES HERE.jpeg
  </h1>
  <p
    className="MuiTypography-root MuiTypography-body1 css-132kt0e-MuiTypography-root"
  >
    I am really good at front end.
  </p>
  <div
    className="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
  >
    <div
      className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <a
        className="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButtonBase-root  css-1knaqv7-MuiButtonBase-root-MuiButton-root"
        href="https://github.com/IThinkThatsKirby"
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        Visit my Git hub HERE
      </a>
    </div>
  </div>
</div>
`;
